### PR TITLE
JP-1645 Supress CRDS unknown reftype errors while retrieving step parameter reference files.

### LINKED ISSUE #5251: Failures to get pars reference files in prefetch cause log messages
_Issue [JP-1645](https://jira.stsci.edu/browse/JP-1645) was created on JIRA by [Robert Jedrzejewski](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rij):_

When step pars reference files are requested from CRDS when running a pipeline, log messages are printed if there is no such pars reference file found in CRDS.  Find a way to disable these messages, as there are many and they are distracting.

### PR DESCRIPTION
**Pull request by stscieisenhamer**
Resolves [JP-1645](https://jira.stsci.edu/browse/JP-1645)
Resolves #5251 

Currently, the CRDS database is sparsely populated with respect to all the possible step parameter reftypes. As such, under normal usage, it is expected to not find a requested reftype. As such, suppress CRDS errors unless debugging is in effect.

### CODE REVIEW

File: jwst/stpipe/tests/test_crds.py [hunk 1]
```diff
@@ -34,6 +36,17 @@ def process(self, input_file):
         return datamodels.DataModel()
 
 
+def test_steppars_error_supression(_jail, caplog):
+    """Test supression of CRDS errors while retrieving step parameters"""
+    model = datamodels.ImageModel((10, 10))
+    model.save('image.fits')
```
Comment (jdavies-st) [inline jwst/stpipe/tests/test_crds.py:42]: This needs to be done in a `with` context manager.  You're leaving an open file here.

**Review by jdavies-st — COMMENTED** [review]
🎉  It will be nice to not see these error messages from CRDS anymore.

Just a minor comment below.

**Review by hbushouse — APPROVED** [review]
Other than what @jdavies-st said, looks OK to me.

File: jwst/stpipe/tests/test_crds.py (hunk 1 above)
Comment (stscieisenhamer) [reply jwst/stpipe/tests/test_crds.py:42]: Will do, but technically no file is open, unless the below is a bug:

```
>>> import jwst.datamodels as dm
>>> m = dm.ImageModel((10, 10))
>>> m.save('junk.fits')
    'junk.fits'
>>> m._files_to_close
    []
```

Comment (jdavies-st) [reply jwst/stpipe/tests/test_crds.py]: Interesting!  I was always under the impression that once something was written, it was then tied to the file.  This a nice, positive thing to discover!

**Review by jdavies-st — APPROVED** [review]

**Comment by stscieisenhamer** [comment]
Changing the status of the PR to `Draft`. Going to pursue a more preferred solution described in [CRDS-381](https://jira.stsci.edu/browse/CRDS-381).

**Comment by stscieisenhamer** [comment]
This PR has been rendered unnecessary by [CCD_783](https://jira.stsci.edu/browse/CCD-783)

**Pull request closed without merging**